src/coreComponents/mesh/DomainPartition.hpp

```diff
@@ -277,6 +277,21 @@ class DomainPartition : public dataRepository::Group
   std::vector< NeighborCommunicator > const & getNeighbors() const
   { return m_neighbors; };
 
+  /**
+   * @brief Get a list of neighbor ranks.
+   * @return Container of neighbor ranks.
+   */
+  std::vector< int > getNeighborRanks() const
```

Comment:
This wasn't meant to be in this PR, but let's just merge it anyway, it's used in follow-up multiscale PR